osu.Game/Beatmaps/BeatmapManager_WorkingBeatmap.cs

```diff
@@ -36,8 +36,9 @@ protected override IBeatmap GetBeatmap()
                     using (var stream = new LineBufferedReader(store.GetStream(getPathForFile(BeatmapInfo.Path))))
                         return Decoder.GetDecoder<Beatmap>(stream).Decode(stream);
                 }
-                catch
+                catch (Exception e)
                 {
+                    Logger.Error(e, "Beatmap failed to load");
```

Comment:
Intentionally using `Logger.Error` here? I'm not against it, but we probably want to change the output target.

Reply:
I think we want to be notified of these errors since they tend to be pretty critical. Keep in mind there already is a secondary catch inside the beatmap decoder itself which catches general parsing errors without reporting to us:

https://github.com/ppy/osu/blob/0f1d30c55ee2c627f680d98a0b284946d7055769/osu.Game/Beatmaps/Formats/LegacyDecoder.cs#L47-L54

Reply:
Okay, let's go with it for now.